CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+Contributing to Dataspace Connector
+===================================
+
+Thank you for your interest contributing to
+the [Eclipse Dataspace Connector](https://projects.eclipse.org/projects/technology.dataspaceconnector)!
+
+## Table of Contents
+
+* [Project Description](#project-description)
+* [Code Of Conduct](#code-of-conduct)
+* [Eclipse Contributor Agreement](#eclipse-contributor-agreement)
+* [How to Contribute](#how-to-contribute)
+    * [Discuss](#discuss)
+    * [Create an Issue](#create-an-issue)
+    * [Submit a Pull Request](#submit-a-pull-request)
+
+## Project Description
+
+See [README.md](README.md) for a comprehensive project description.
+
+## Code Of Conduct
+
+This project is governed by
+the [Eclipse Code Of Conduct](https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php). You are expected to
+keep up to the code of conduct while participating in this project.
+
+## Eclipse Contributor Agreement
+
+Before your contribution can be accepted by the project, you need to create and electronically sign
+a [Eclipse Contributor Agreement (ECA)](http://www.eclipse.org/legal/ecafaq.php):
+
+1. Log in to the [Eclipse foundation website](https://accounts.eclipse.org/user/login/). You will need to create an
+   account with the Eclipse Foundation if you have not already done so.
+2. Click on "Eclipse ECA", and complete the form.
+
+Be sure to use the same email address in your Eclipse account that you intend to use when you commit to GitHub.
+
+## How to Contribute
+
+### Discuss
+
+If you have questions or suggestions feel free to contact the project developers via
+the [project's "dev" list](https://dev.eclipse.org/mailman/listinfo/dataspaceconnector-dev). If you suppose there is a
+bug or issue contribute to discussions in
+[existing issues](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues?q=is%3Aissue+is%3Aopen)
+otherwise [create a new issue](#create-an-issue).
+
+### Create an Issue
+
+Although your contributions are always welcome please remember to search
```

Comment:
Important to mention that people should search for existing issues before opening a new one but I wouldn't start with that disclaimer. First of all we embrace every idea + contribution.

Reply:
You're right - wording was perhaps a bit harsh. Did some slight rewording. Please let me know whether it's better now.